lib/get-custom-properties.js

```diff
@@ -47,11 +87,28 @@ const rootSelectorRegExp = /^:root$/i;
 const customPropertyRegExp = /^--[A-z][\w-]*$/;
 
 // whether the node is an html or :root rule
-const isHtmlRule = node => node.type === 'rule' && htmlSelectorRegExp.test(node.selector) && Object(node.nodes).length;
-const isRootRule = node => node.type === 'rule' && rootSelectorRegExp.test(node.selector) && Object(node.nodes).length;
+const isHtmlRule = node =>
+	node.type === "rule" &&
+	htmlSelectorRegExp.test(node.selector) &&
+	Object(node.nodes).length;
+const isRootRule = node => {
+	return (
+		node.type === "rule" &&
+		rootSelectorRegExp.test(node.selector) &&
+		Object(node.nodes).length
+	);
+};
+const isMediaQuery = node => {
```

Comment:
For example, right here I almost miss your change because it’s caught up in the formatting changes you’ve made to the proceeding rule.